### ISSUE TITLE
Missing entities for some vehicles, extra info for others, after v5 API update 

### ISSUE THREAD

**Issue by GeneralLouis** [itchannel/fordpass-ha#375]
Once again I want to thank you for the great and quick work on the updates when Ford API team moves your cheese.

I am re-writing all my frontend and customization but I noticed some changes, 1 entry reports all 4 tire pressures now too (Explorer 2021), and another vehicle now says tires are not supported (2017 Escape), do you have a list of supported features? Maybe one by vehicle? Or is there a way I can go and fetch this and share with everyone instead? (Also the odometer is missing on the 2017 escape)

I don't want to submit an issue here if it's just taking a while for all the kinks to get worked out, I've seen some items drop off only to come back later on.

**Comment by SquidBytes** [comment]
> do you have a list of supported features? Maybe one by vehicle? Or is there a way I can go and fetch this and share with everyone instead? 

Yes, vehicle variety has been quite the challenge. If you would like to share, or see your vehicles data used to generate sensors you can open the `autonomicData.py` script, add the information, and then run it in a terminal `python3 autonomicData.py`.
It will automatically redact personal information and GPS coordinates so you can share it.

> (Also the odometer is missing on the 2017 escape)

This is a bug we know about when we changed over to using HomeAssistant for conversions. You can fix it by replacing your `sensor.py` file with the `sensor.py` file in the [1.58 Branch Here](https://github.com/itchannel/fordpass-ha/tree/1.58)

**Comment by itchannel** [comment]
@GeneralLouis Please try 1.58 for your odometer issue. As for what vehicles support what if you could do what @SquidBytes mentions it will help us correlate data on models.

**Comment by GeneralLouis** [comment]
I saw 1.58 was just released and I updated it (with restart), still no odometer, but I will give it some time to see if appears.

I will get you the vehicle data as soon as I can!

**Comment by SquidBytes** [comment]
> could we add an opt-in option to use this script to anonymously send vehicle data to the community so we can get a catalog of supported features? I know that even different trims will vary what options are supported, so it may help lower issues if users can search for their build and know if it's a bug or lack of support by Fordpass.

Catalog of supported features is a very large challenge we are facing. It is a goal we would like to display somewhere. We have a to-do improvement for the `autonomicData.py` script to also return vehicle capabilities so we can start cataloging those. I have been collecting the json data people have been sharing, but to setup an automatic share is a good idea - I'm just not sure how that would be implemented.
